Library/Homebrew/sandbox.rb

```diff
@@ -91,24 +98,36 @@ def exec(*args)
     @start = Time.now
     safe_system SANDBOX_EXEC, "-f", seatbelt.path, *args
   rescue
-    if ARGV.verbose?
-      ohai "Sandbox profile:"
-      puts @profile.dump
-    end
+    @failed = true
     raise
   ensure
     seatbelt.unlink
-    unless @log.nil?
+    if @log
       sleep 0.1 # wait for a bit to let syslog catch up the latest events.
       syslog_args = %W[
-        -F '$((Time)(local))\ $(Sender)[$(PID)]:\ $Message'
+        -F $((Time)(local))\ $(Sender)[$(PID)]:\ $(Message)
         -k Time ge #{@start.to_i}
+        -k Message S deny
         -k Sender kernel
         -o
         -k Time ge #{@start.to_i}
+        -k Message S deny
         -k Sender sandboxd
       ]
-      quiet_system "syslog #{syslog_args * " "} | grep deny > #{@log}"
+      logs = Utils.popen_read("syslog", *syslog_args)
+      unless logs.empty?
+        @log.write Sandbox.log_warning_message
+        @log.write logs
```

Comment:
You mean the log format? 

Currently it's `$((Time)(local)) $(Sender)[$(PID)]: $(Message)`. e.g. `Aug 27 21:19:32 sandboxd[295]: ruby(93659) deny file-write-create /1`. We can tweak it if you want.